docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -0,0 +1,152 @@
+---
+title: How to create example cases for a Custom Expectation
+---
+import Prerequisites from '../creating_custom_expectations/components/prerequisites.jsx'
+
+This guide will help you add example cases to document and test the behavior of your [Expectation](../../../reference/expectations/expectations.md). 
+
+<Prerequisites>
+
+ - Created a [Custom Expectation](../creating_custom_expectations/overview.md)
+
+</Prerequisites>
+
+
+Example cases in Great Expectations serve a dual purpose:
+* They help the users of the Expectation understand its logic by providing examples of input data that the Expectation will evaluate;
+* And provide test cases that the Great Expectations testing framework can execute automatically.
```

Comment:
```suggestion
Example cases in Great Expectations serve a dual purpose:
* First, they help the users of the Expectation understand its logic by providing examples of input data that the Expectation will evaluate.
* Second, they provide test cases that the Great Expectations testing framework can execute automatically.
```
Since you state a dual purpose, associating each item with a number can help people keep things mentally sorted.  Likewise, the connection between the two items is implied by their grouping in the list -- you can just treat each as a full sentence, instead of using semicolon list notation, which is more formal than our style guide calls for.